pkg/machine/qemu/machine.go

```diff
@@ -432,6 +432,9 @@ func (v *MachineVM) Remove(name string, opts machine.RemoveOptions) (string, fun
 		return "", nil, errors.Errorf("running vm %q cannot be destroyed", v.Name)
 	}
 
+	// Get path to socket and pidFile before we do any cleanups
+	qemuSocketFile, pidFile, errSocketFile := v.getSocketandPid()
+
```

Comment:
Why do you call it here and only use it 30 lines later, please move this down.

Reply:
@Luap99 I thought rumtime would be removed and become invalid after `RemoveConnection` but just checked everything is static so cool :) Fixed.